### PR TITLE
added getter for contract definitions

### PR DESCRIPTION
**Pull request by RosieMurphy0**
Signed-off-by: RosieMurphy0 <rosie.murphy@ibm.com>

Adds a function that returns the array that holds the contracts for a specified channel from the config file. 

Contributes to #940

### CODE REVIEW

File: packages/caliper-fabric/lib/ConnectorConfiguration.js [hunk 1]
```diff
@@ -86,6 +86,17 @@ class ConnectorConfiguration {
...
         return null;
     }
+    /**
+     * return array of contract definitions for specific channel
+     * @param {string} channelPassed channel wanted
+     * @returns {Array} all of contract definitions for channel
+    */
+    getContractDefinitionsForChannelName(channelPassed) {
+        const channelList = this.adapterConfiguration.channels;
+        const channelWanted = channelList.filter(channelContracts => channelContracts.channelName === channelPassed);
```
Comment (davidkel) [inline packages/caliper-fabric/lib/ConnectorConfiguration.js:96]: As we don't have a validator yet and not sure when a validator would be made available for now lets add some defensive checks here to make sure that there is a channelList, so you can check the example used in getChannelNamesForCreation.

File: packages/caliper-fabric/lib/ConnectorConfiguration.js [hunk 2]
```diff
@@ -86,6 +86,17 @@ class ConnectorConfiguration {
 
         return null;
     }
+    /**
+     * return array of contract definitions for specific channel
+     * @param {string} channelPassed channel wanted
+     * @returns {Array} all of contract definitions for channel
+    */
+    getContractDefinitionsForChannelName(channelPassed) {
```
Comment (davidkel) [inline packages/caliper-fabric/lib/ConnectorConfiguration.js:94]: would suggest changing `channelPassed` to `channelName` for the parameter

File: packages/caliper-fabric/lib/ConnectorConfiguration.js [hunk 3]
```diff
@@ -86,6 +86,17 @@ class ConnectorConfiguration {
...
+        const channelList = this.adapterConfiguration.channels;
+        const channelWanted = channelList.filter(channelContracts => channelContracts.channelName === channelPassed);
+        const contractDefinitions = channelWanted[0].contracts;
```
Comment (davidkel) [inline packages/caliper-fabric/lib/ConnectorConfiguration.js:97]: For consistency, if the contracts definition is not there, then we should just return an empty array. Currently if it wasn't present we would be returning null or undefined. This will need a further test to check for.

File: packages/caliper-fabric/lib/ConnectorConfiguration.js [hunk 4]
```diff
@@ -86,6 +86,22 @@ class ConnectorConfiguration {
...
+     * return array of contract definitions for specific channel
+     * @param {string} channelName channel name wanted
+     * @returns {Array} all of contract definitions for channel
+    */
+    getContractDefinitionsForChannelName(channelName) {
+        const channelList = this.adapterConfiguration.channels;
+
+        if (channelList && Array.isArray(channelList)){
+            const channelWanted = channelList.filter(channelContracts => channelContracts.channelName === channelName);
+            const contractDefinitions = channelWanted[0].contracts;
```
Comment (davidkel) [inline packages/caliper-fabric/lib/ConnectorConfiguration.js:99]: Should test here what is in contracts, eg
```
if (contractDefinitions && 
    Array.isArray(contractDefinitions) {
   return contractDefinitions;
}
```
Then add a test to see that you get an empty array is the contracts property hasn't been specified

**Review by davidkel — APPROVED** [review]
LGTM
